src/testapi_cube.c

```diff
@@ -0,0 +1,38 @@
+#include "gmt_dev.h"
```

Comment:
Is it OK to use `gmt.h` instead of `gmt_dev.h`?
```suggestion
#include "gmt.h"
```

Reply:
Yes, committed separately since another change was needed as well.